services/construction.go

```diff
@@ -127,7 +129,12 @@ func (s *constructionAPIService) ConstructionSubmit(
 
 	if err := s.oasisClient.SubmitTxNoWait(ctx, request.SignedTransaction); err != nil {
 		loggerCons.Error("ConstructionSubmit: SubmitTxNoWait failed", "err", err)
-		return nil, ErrUnableToSubmitTx
+		// TODO: Have oasis-core use a distinct error code for this.
```

Comment:
Please file an issue, it should be trivial to support a proper error code.

Reply:
Filed as https://github.com/oasisprotocol/oasis-core/issues/3192 implemented in https://github.com/oasisprotocol/oasis-core/pull/3193.

Reply:
changed it to that